_posts/2021-02-16-cockpit-238.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Cockpit 238
+author: kkoukiou
+date: 2021-02-17
+tags: cockpit linux
+slug: cockpit-238
+category: release
+summary: Cockpit with restyled updates page
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 238.
+
+### Updates: Show the list of services and applications that are outdated
```

Comment:
Perhaps something like:

```suggestion
### Updates: List outdated software that needs a restart
```

Reply:
This is a bit vague about what restart meant, reboot or restart service. Therefore I kept it out of the header.

Reply:
Right, but the point is to restart the software when needed, not show a list.